README.md

```diff
@@ -66,6 +66,33 @@ env:
   CI_JOB_URL: ${{ github.event.repository.html_url }}/actions/runs/${{ github.run_id }}
 ```
 
+## Customise deploy messages
+
+You can choose to customise the deploy messages globally or per project. If you set both, the project config will take precedence. For each task (deploy start or deploy success) you can customise two elements: `text` and `colour`.
+
+To set globally, in `group_vars/all/helpers.yml`:
+
+```yaml
+slack_notify:
```

Comment:
I needed to make this `slack_notify_config` to match refs used in deploy_start/success.yml. Not sure if that's an issue my side?

Tried these:
![2025-05-07 1415 - Code@2x](https://github.com/user-attachments/assets/79da061d-77ad-4af3-9dec-b144fbff74a7)
![2025-05-07 1416 - Code@2x](https://github.com/user-attachments/assets/0ef84aa4-a15b-4049-8b5f-104258b0ff41)

Get this for both:
![2025-05-07 1416 - iTerm2](https://github.com/user-attachments/assets/88d36e83-b9f6-4556-a29d-dab28f89a6c8)

Sends the msg successfully when I update the key to: `slack_notify_config`
![2025-05-07 1418 - Code@2x](https://github.com/user-attachments/assets/50f7ae54-f1ad-4e78-97c7-ea51dfe21be8)

Reply:
I have this: 
<img width="460" alt="Screenshot 2025-05-08 at 9 53 30 AM" src="https://github.com/user-attachments/assets/0e68cbf2-93d8-49e9-a6df-f4a87fe7d403" />

The error you're getting is more likely to do with the var task not being picked up, as that is where `slack_notify_config` is actually set. In your final screenshot, that would work because you're now setting the config, but you wouldn't then be able to override that per project. 

tl;dr this issue is a knock-on effect of the Load vars task failing.